src/Ship.cpp

```diff
@@ -213,6 +218,8 @@ void Ship::InitMaterials()
 
 void Ship::Init()
 {
+	Output("Running Ship::Init()\n");
```

Comment:
I'm assuming this is a leftover debug printout? In any case, I'd rather not have it, as it clutters the output without adding much information.